apps/tx-builder/src/utils.ts

```diff
@@ -19,45 +19,6 @@ export const enum FETCH_STATUS {
   SUCCESS = 'SUCCESS',
   ERROR = 'ERROR',
 }
-
-export enum CHAINS {
```

Comment:
it uses safe-apps-provider for RPC calls now

Reply:
are you sure that we can remove it?

the ens resolution is not working for me:

![Captura de pantalla 2022-07-18 a las 14 52 04](https://user-images.githubusercontent.com/26763673/179515350-8cef1a89-3b8f-4cf6-9ea7-e86d60cedf00.png)

Reply:
Turned out to be a bug with the `AddressInput`, for some reason it only does ens resolving when you change passed `address` value instead of reacting to users input 🤷 good catch senor